doc/sphinxext/move_gallery_links.py

```diff
@@ -0,0 +1,171 @@
+from pathlib import Path
```

Comment:
I don't really understand what this is doing at all. And I'm not sure which problem exactly it's trying to solve. A top comment on this file would be nice.

Reply:
I think a top level comment that explains the purpose of the script is indeed needed (while the details are explained in the inline comments).